src/main/java/edu/wpi/first/shuffleboard/widget/WidgetType.java

```diff
@@ -7,11 +7,15 @@
 
 public interface WidgetType extends Supplier<Widget> {
   /**
+   * Get the name of the widget.
+   *
```

Comment:
this is way too verbose, and just clutters up the code for no reason. Either remove the `@returns` statement or leave it as the only doc.